### DISCUSSION TITLE
How do I disable mailhog?

### DISCUSSION THREAD

**Discussion by p11v** [markshust/docker-magento#449] [Q&A] [answered]
It's not in docker-compose.yml.

**Comment by rangerz** [comment, marked as answer]
mailhog service is in `docker-compose.dev.yml`

The email setting is in [/usr/local/etc/php/php.ini](https://github.com/markshust/docker-magento/blob/master/images/php/7.4/conf/php.ini). You may also need to know.
```
sendmail_path = "/usr/local/bin/mhsendmail --smtp-addr=mailhog:1025"
```

**Reply by p11v** [reply]
Got it. Thanks.
